filebeat/tests/integration/systemlogs_test.go

```diff
@@ -63,6 +68,83 @@ func TestSystemLogsCanUseJournaldInput(t *testing.T) {
 		"journalctl started with PID",
 		10*time.Second,
 		"system-logs did not start journald input")
+
+	// Scan every event in the output until at least one from
+	// each fileset (auth, syslog) is found.
+	waitForAllFilesets(
+		t,
+		filepath.Join(workDir, "output*.ndjson"),
+		"did not find events from both filesets: 'auth' and 'syslog'",
+	)
+}
+
+func waitForAllFilesets(t *testing.T, outputGlob string, msgAndArgs ...any) {
+	require.Eventually(
+		t,
+		findFilesetNames(t, outputGlob),
+		time.Minute,
+		10*time.Millisecond,
+		msgAndArgs...)
+}
+
+func findFilesetNames(t *testing.T, outputGlob string) func() bool {
+	f := func() bool {
+		files, err := filepath.Glob(outputGlob)
+		if err != nil {
+			t.Fatalf("cannot get files list for glob '%s': '%s'", outputGlob, err)
+		}
+
+		if len(files) > 1 {
+			t.Fatalf(
+				"only a single output file is supported, found: %d. Files: %s",
+				len(files),
+				files,
+			)
+		}
+
+		foundSyslog := false
+		foundAuth := false
+
+		file, err := os.Open(files[0])
```

Comment:
The linter says the `err` value is not used below.

Reply:
Thanks!